_data/health.yml

```diff
@@ -207,6 +207,18 @@ websites:
       othercrypto: no
       doc: https://www.shinyleaf.com/promotions/bitcoin-cash-payment/
 
+    - name: Smile Acapulco
+      url: http://www.smileacapulco.com/
```

Comment:
Take a look over our documentation on the lang tag. I want to start adding listings with the lang tag that we know are definitely a single language like this one.

We dont use it yet but it will be useful in the future and having things that we add now include it gives us a good subset of data we can make immediate use of when we refactor to use it. Though I think we might use lang to determine alternative tweet text already.

Reply:
Yeah, I will read about the lang tag

Reply:
Added `lang: es` key

Reply:
Thank you!